pkg/handler/server_ssh.go

```diff
@@ -412,7 +412,7 @@ func (s *Server) proxyAssetCommand(sess ssh.Session, sshClient *srvconn.SSHClien
 	}
 	ctx, cancel := context.WithCancel(sess.Context())
 	defer cancel()
-
+	respSession.TokenId = tokenInfo.Id
 	traceSession := session.NewSession(&respSession, func(task *model.TerminalTask) error {
 		switch task.Name {
 		case model.TaskKillSession:
```

Comment:
There is no need to check these lines directly because they seem fine according to their current status. The code does not have any known inaccuracies at this point, including syntax or functionality issues.

As the function is defined without specific constraints, we should assume that it will be used as written and perform regular checks on its execution throughout the server's lifecycle.

However, you can suggest improvements based on best practices, but there won't necessarily be any inconsistencies between this particular line:

```
ctx, cancel := context.WithCancel(sess.Context())
```

because adding a condition like `if !sess.IsActive()` would already handle all cases if sessions are validly started during initialization within Server itself.